posts/2024-07-02-24.0.0.7-beta.adoc

```diff
@@ -0,0 +1,243 @@
+
+The new features, `restfulWS-4.0` and `restfulWSClient-4.0`, are primarily aimed at application developers. This release brings incremental improvements that are made to Jakarta RESTful Web Services provided in the 4.0 release of that Jakarta project. It provides new convenience API methods:
+
+- The `containsHeaderString()` method has been added to `ContainerRequestContext`, `ContainerResponseContext`, and `HttpHeaders` to help determine whether a header contains a specified value.
+- The `UriInfo.getMatchedResourceTemplate()` method is a new addition that returns a full template for a request.
+- New `MediaType` values have been introduced, including `APPLICATION_JSON_PATCH_JSON` and `APPLICATION_MERGE_PATCH_JSON`, to handle specific media types.
+
+These updates aim to streamline the process of developing RESTful services. To implement these new features, add the following to your `server.xml` file.
+
+For the RESTful web services feature.
+[source,xml]
+----
+<featureManager>
+    <feature>restfulWS-4.0</feature>
+</featureManager>
+----
+
+For the RESTful web services client feature.
+[source,xml]
+----
+<featureManager>
+    <feature>restfulWSClient-4.0</feature>
+</featureManager>
+----
+
+For more information, see the Jakarta RESTful Web Services 4.0 link:https://jakarta.ee/specifications/restful-ws/4.0/[specification] and the link:https://jakarta.ee/specifications/restful-ws/4.0/apidocs/jakarta.ws.rs/module-summary[Javadoc].
+
+    
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28693
+// Contact/Reviewer: Channyboy
+// // // // // // // // 
+[#monitor10]
+== Enhanced HTTP Request Monitoring with Monitor 1.0
+
+Open Liberty introduces a significant update that enhances the tracking of HTTP connections that are made to the server. With the Monitor-1.0 feature (`monitor-1.0`) and any JEE10 feature that uses the servlet engine, Open Liberty can now record detailed information about HTTP requests into an `HttpStatsMXBean`. This includes the request method, response status, duration, HTTP route, and other attributes that align with the Open Telemetry HTTP metric semantic convention.
+
+If `mpMetrics-5.0` or `mpMetrics-5.1` is also active, Open Liberty creates corresponding HTTP metrics, such as `http.server.request.duration`, using a Timer type metric. To use these metrics effectively, it is essential to configure histogram buckets explicitly for `Timer` and `Histogram` type metrics by using the MicroProfile Metrics. For more details, see the `mp.metrics.distribution.histogram.buckets` property on the link:https://openliberty.io/docs/latest/microprofile-config-properties.html#metrics[MicroProfile Config Properties page].
```

Comment:
Just only need to configure `Timer` metric type (since the metric is a Timer).
Consequently the property would be the `mp.metrics.distribution.timer.buckets` for the Timer.

I realized i used the  `mp.metrics.distribution.histogram.buckets` one in the issue, sorry!